rbac/management/utils.py

```diff
@@ -50,12 +52,28 @@
 
 logger = logging.getLogger(__name__)
 
+# Configure OAuth credentials with direct token URL
+inventory_auth_credentials = OAuth2ClientCredentials(
+    client_id=settings.INVENTORY_API_CLIENT_ID,
+    client_secret=settings.INVENTORY_API_CLIENT_SECRET,
+    token_endpoint=settings.INVENTORY_API_TOKEN_URL,  # Direct token endpoint
+)
+
+call_credentials = oauth2_call_credentials(inventory_auth_credentials)
```

Comment:
56-76: This is same code both places, is it possible to consolidate it into one ?